autopush/router/simple.py

```diff
@@ -76,6 +75,9 @@ def route_notification(self, notification, uaid_data):
         # Determine if they're connected at the moment
         node_id = uaid_data.get("node_id")
         uaid = uaid_data["uaid"]
+        self.udp = None
+        if "udp" in uaid_data:
+            self.udp = uaid_data["udp"]
```

Comment:
If "udp" is in uaid_data, would accessing "udp" ever trigger a key error? (honest question, I'd love to be sure about something like this.)